recipes/default.rb

```diff
@@ -26,14 +26,26 @@
 mvn_version = node['maven']['version'].to_s
 
 ark 'maven' do
-  url      node['maven'][mvn_version]['url']
-  checksum node['maven'][mvn_version]['checksum']
-  home_dir node['maven']['m2_home']
-  version  node['maven'][mvn_version]['version']
+  url             node['maven'][mvn_version]['url']
+  checksum        node['maven'][mvn_version]['checksum']
+  home_dir        node['maven']['m2_home']
+  win_install_dir node['maven']['m2_home']
```

Comment:
`win_install_dir` wasn't added until [ark v9.0](https://github.com/burtlo/ark/commit/12f9b4835a54c78ea57e35d5be33193fa7451f85). This cookbook still relies on ark v4.0 and as a result, users get an error about an undefined method `win_install_dir`: #54.